cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableDefinition.java

```diff
@@ -88,7 +92,10 @@ private HTableDescriptor getHTableDescriptor(DatasetSpecification spec) throws I
 
     final HTableDescriptor tableDescriptor = new HTableDescriptor(tableName);
     tableDescriptor.addFamily(columnDescriptor);
-    tableDescriptor.addCoprocessor(hBaseTableUtil.getIncrementHandlerClassForVersion().getName());
+    AbstractHBaseDataSetAdmin.CoprocessorJar cpJar =
+      HBaseOrderedTableAdmin.createCoprocessorJarInternal(conf, locationFactory, hBaseTableUtil, true);
+    tableDescriptor.addCoprocessor(hBaseTableUtil.getIncrementHandlerClassForVersion().getName(),
```

Comment:
yes: CDAP-1193 0 it is mentioned in todos in this change. That will be long term fix